code/game/machinery/spaceheater.dm

```diff
@@ -3,8 +3,8 @@
 	density = 1
 	icon = 'icons/obj/atmos.dmi'
 	icon_state = "sheater0"
-	name = "space heater"
-	desc = "Made by Space Amish using traditional space techniques, this heater is guaranteed not to set the station on fire."
+	name = "Space A/C unit"
```

Comment:
This should all be lowercase. It's proper naming format to make it lowercase because of how noun detection works. In other words, it should be ```space A/C unit```.

Reply:
I did a update now, with all lowercase